mlrun/alerts/alert.py

```diff
@@ -16,7 +16,6 @@
 
 import mlrun
 import mlrun.common.schemas.alert as alert_constants
```

Comment:
Out of curiosity - why do you call this `alert_constants`? There are the schemas of the alert types, isn't it? Why not call it `alert_types` or `alert_objects` or something like this?
It looks strange when you use `alert_constants.AlertNotification` and I think "but this is not a constant!".